src/netspresso_trainer/dataloaders/classification/transforms.py

```diff
@@ -1,21 +1,29 @@
+import inspect
```

Comment:
Using `inspect` ....?

Ok... but then this PR would be one of the biggest changes during our patch.

Reply:
If we start to use `inspect` here, we could also use it when calling model instance.